app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,27 @@
+ module SessionsHelper
+  # Logs in the given user.
+  def log_in(user)
+    session[:user_id] = user.email_hash
```

Comment:
Should this be:

`session[:user_id] = user.id`

?

Reply:
Yeah I'll update that